pkg/tfgen/examples_coverage_exporter.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2016-2021, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file implements the methods used by the Coverage Tracker in order
+// to export the data it collected into various JSON formats.
+
+package tfgen
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+// The export utility's main structure, where it stores the desired output directory
+// and a reference to the CoverageTracker that created it
+type CoverageExportUtil struct {
```

Comment:
Does this type need to be exported? If we do not intend to consume it from other packages, I suggest we unexport it by changing its name to `coverageExportUtil`.